crane/app_util.py

```diff
@@ -107,9 +114,16 @@ def wrapper(image_id, *args, **kwargs):
             if not repo_tuple.protected:
                 found_match = True
                 break
-            elif cert and cert.check_path(repo_tuple.url_path):
-                found_match = True
-                break
+            elif repo_tuple.protected:
+                value = repo_tuple.url_path
+                logger.info('path before check for images %s' % value)
+                if 'webassets/docker' in value:
+                    value = value[len('webassets/docker') + 1:]
```

Comment:
Should the checked-for prefix be 'webassets/docker/' instead, including the trailing slash? Otherwise it is not clear why 1 is being added to the length here.

Reply:
value generated is ''/webassets/docker/content/dist/rhel/server/7/7Server/x86_64/containers/registry/'' so +1 was added to escape starting "/" from value.

Let me make it value.startswith('/webassets/docker') and value = value[len('/webassets/docker'):]